### PR TITLE
Fix label_sync file and README

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

There was some hidden indentation problems in the yaml file 🤕.
Also updated the README with some link and name fixes.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @dlorenc @bobcatfish @abayer 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by vdemeester** [comment]
(it is deployed too)

File: label_sync/README.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ We currently run a cron job synchronize labels in
 We use a separate cluster in a restricted project because modifying the labels requires write permission on all repos.
 
 We have a CronJob to sync the labels, defined
-[here](https://github.com/tekton/testing/blob/master/label_sync/cluster/label_sync_job.yaml).
+[here](https://github.com/tektoncd/plumbing/blob/master/label_sync/cluster/label_sync_job.yaml).
 After making changes to `labels.yaml`, we need to update the configmap
-[label-config-v2](https://github.com/tekton/testing/blob/master/label_sync/cluster/label_sync_job.yaml#L37):
+[label-config-v2](https://github.com/tektoncd/plumbing/blob/master/label_sync/cluster/label_sync_job.yaml#L37):
```
Comment (bobcatfish) [inline label_sync/README.md]: :sweat_smile:

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -338,13 +338,13 @@ default:
...
-      description: Issues related to user-studies
-      name: user-study
-      target: issues
-      addedBy: humans
+    labels:
+      - color: 3f6eb5
+        description: Issues related to user-studies
+        name: user-study
+        target: issues
+        addedBy: humans
```
Comment (bobcatfish) [inline label_sync/labels.yaml]: question - does this mean you are doing user studies @vdemeester ? we've got an `area/user-study` label in pipelines also (https://github.com/tektoncd/pipeline/labels/area%2Fuser-study) but i dont think we actually need it anymore

File: label_sync/labels.yaml [hunk 3]
```diff
@@ -353,50 +353,56 @@ repos:
...
+      - color: 160a72
+        description: This issue needs some help during design phase
+        name: design-help-wanted
+        target: issues
+        addedBy: humans
   tektoncd/pipeline:
+    labels:
+      - color: ffcdc4
+        description: For consideration when planning the next milestone
+        name: maybe-next-milestone
```
Comment (bobcatfish) [inline label_sync/labels.yaml]: :innocent:

File: label_sync/labels.yaml [hunk 4]
```diff
@@ -353,50 +353,56 @@ repos:
...
+      - color: 3E4B9E
+        description: Issues that should be considered as Epics (aka multiple sub-tasks, …)
+        name: Epic
+        target: both
+        addedBy: humans
+      - color: 64e5ba
+        description: This is for some internal Google project tracking
+        name: okr
+        target: both
+        addedBy: humans
```
Comment (bobcatfish) [inline label_sync/labels.yaml]: i wish we could have a list we share between all the projects without defining it so many times

File: label_sync/labels.yaml (hunk 2 above)
Comment (vdemeester) [reply label_sync/labels.yaml]: Ahh 😅 as it was there before I kept it 😇

File: label_sync/labels.yaml (hunk 4 above)
Comment (vdemeester) [reply label_sync/labels.yaml]: There is one (the default). If we move this one in there, all the repository will have it. I wasn’t sure about keeping it only for pipeline or for all

File: label_sync/labels.yaml (hunk 2 above)
Comment (bobcatfish) [reply label_sync/labels.yaml]: ah kk ill make a note and see if can get around to removing them XD

File: label_sync/labels.yaml (hunk 4 above)
Comment (bobcatfish) [reply label_sync/labels.yaml]: ah kk! didnt know we could do that :D

**Pull request merged**